screenpy_selenium/resolutions/custom_matchers/is_invisible_element.py

```diff
@@ -0,0 +1,46 @@
+"""
+A matcher that matches an invisible element. For example:
+
+    assert_that(driver.find_element_by_id("search"), is_invisible_element())
+"""
+
+
```

Comment:
This is the tiniest nitpick i think i've evern included in a PR review, but...

Could you make sure the number of blank lines between the docstring and the first import is consistent? My preference is one blank line. :P

Reply:
Sure. Doesn't black handle that automatically?

Reply:
It does if you run it or have `pre-commit` installed!

Reply:
Looks like black does _not_ care about spacing between module docstring and the imports.  We'll just have to catch those in code review.